packages/schema/src/query_response.rs

```diff
@@ -0,0 +1,9 @@
+use std::collections::BTreeMap;
+
+use schemars::schema::RootSchema;
+
+pub use cosmwasm_schema_derive::QueryResponses;
+
+pub trait QueryResponses {
+    fn query_responses() -> BTreeMap<&'static str, RootSchema>;
```

Comment:
`QueryMsg::query_responses()` is a bit redundant. What about `QueryMsg::response_types()` or `QueryMsg::response_schemas()`?